File: Modul-3/Soal-2/PRAK302-2410817210008-AhmadUlyani.py
n = int(input())

if 80 <= n <= 100:
    nilai = "A"
elif 70 <= n < 80:
    nilai = "B"
elif 60 <= n < 70:
    nilai = "C"
elif 50 <= n < 60:
    nilai = "D"
elif 0 <= n < 50:
    nilai = "E"
else:
    nilai = "Nilai tidak valid"

print(f"{nilai}")

File: Modul-3/Soal-2/PRAK302-2410817210008-AhmadUlyani.c
#include <stdio.h>
int main(){
    int n;
    scanf("%d", &n);
    if (n>=80 && n<=100){
        printf("A");
    }
    else if(n>=70 && n<80){
        printf("B");
    }
    else if(n>=60 && n<70){
        printf("C");
    }
    else if(n>=50 && n<60){
        printf("D");
    }
    else if(n>=0 && n<50){
        printf("E");
    }
    else{
        printf("Nilai tidak valid");
    }

    return 0;
}